acceptance/acceptance_test.go

```diff
@@ -72,6 +72,14 @@ func TestReleases(t *testing.T) {
 	testscript.Run(t, testScriptParamsFor(tsEnv, "release"))
 }
 
+func TestRepos(t *testing.T) {
+	var tsEnv testScriptEnv
+	if err := tsEnv.fromEnv(); err != nil {
+		t.Fatal(err)
+	}
+	testscript.Run(t, testScriptParamsFor(tsEnv, "repos"))
```

Comment:
Oh sorry, one more minor thing for consistencies sake, we've been naming the dir after the command i.e. `repo`.

Reply:
Gah, you're right... Total oversight on my part 🤦 Definitely gonna make this change